modules/videoio/src/cap_android_camera.cpp

```diff
@@ -304,16 +304,16 @@ class AndroidCameraCapture : public IVideoCapture
         AImage_getPlaneRowStride(image.get(), 0, &yStride);
         AImage_getPlaneRowStride(image.get(), 1, &uvStride);
         AImage_getPlaneData(image.get(), 0, &yPixel, &yLen);
-        AImage_getPlaneData(image.get(), 1, &vPixel, &vLen);
-        AImage_getPlaneData(image.get(), 2, &uPixel, &uLen);
+        AImage_getPlaneData(image.get(), 1, &uPixel, &vLen);
+        AImage_getPlaneData(image.get(), 2, &vPixel, &uLen);
```

Comment:
Don't you also want to swap `vLen` with `uLen` ?

Reply:
yeah, I missed that. They were of same length, so I did not catch it.

Reply:
@komakai I made the changes. Can please suggest how to take this PR further? I am not sure why the `default` build has failed. I can probably initiate a retest is that is something holding us back.

Reply:
@saikatnanda I'm also not sure why the build is failing - will try and take a look when I get a minute. In the meantime could you rollup your changes into a single commit? Also it might be worth rebasing onto the latest `master` - sometimes that solves weird build failures.

Reply:
Thanks. I will then rebase and will create a fresh PR to look better.